test/synth_simulation_workspace.py

```diff
@@ -125,7 +125,23 @@ def calculate(config, seed):
         if np.isnan(fine_result) or np.isnan(coarse_result):
             raise Exception("result is nan")
 
-        return fine_result, coarse_result
+        quantity_format = SynthSimulationWorkspace.result_format()
```

Comment:
I propose to have both synth_simulation and synth_simulation_wrokspace in a single file, and reuse their common code. Remove or rename synth_simulation_test as that is expected to be a test source file by the pytest.